tools/update_x86_cpu_model_enums.zig

```diff
@@ -0,0 +1,196 @@
+const std = @import("std");
```

Comment:
make sure that this tool is ci tested in test/standalone.zig:63.

Reply:
https://github.com/ziglang/zig/pull/18193/commits/2841c9e61916309884093544f749ece6afb67d5f